src/trans-netlist/trans_to_netlist.cpp

```diff
@@ -142,6 +146,26 @@ class convert_trans_to_netlistt:public messaget
 
 /*******************************************************************\
 
+Function: has_next_symbol
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+static bool has_next_symbol(const exprt &expr)
+{
+  auto begin = expr.depth_cbegin(), end = expr.depth_cend();
+  return std::find_if(begin, end, [](const exprt &expr) {
+           return expr.id() == ID_next_symbol;
+         }) != end;
+}
+
+/*******************************************************************\
+
```

Comment:
Use `has_subexpr(expr, ID_next_symbol)` from `expr_utils.{h,cpp}` (which does exactly the above).